admin/controllers/school.js

```diff
@@ -37,7 +41,7 @@ const getPupils = async (req, res, next) => {
   res.locals.sortColumn = sortColumn || 'lastName'
   const order = JSON.parse(sortOrder)
   res.locals.sortOrder = typeof order === 'boolean' ? !order : true
-  res.locals.sortClass = order === false ? 'triangle down' : 'triangle'
+  res.locals.sortClass = order === false ? 'sort up' : 'sort'
```

Comment:
Change discussed with @george to consolidate code around sorting functionality.

Reply:
@pris54 I think you@mentioned the wrong George :)

Reply:
@george indeed, apologies!